Lad monitor

dterror-zz opened the pull request:
This adds a new job that will check for late-arriving-data[1] in our Camus drops and will fail (and alert) if it finds any case.

[1] we're defining late-arriving-data as data (not metadata) that has been written after the `_IMPORTED` flag has been added to the folder.

@drdee

dterror commented on camus-shopify/script/update_schedule:
```diff
@@ -68,7 +73,12 @@ def main():
                                             'dependencies': 'Camus Import'}
                                            ))
-    project.add_job("Camus", Job({'dependencies': 'Camus Watermark', 'type':'noop'}))
+    project.add_job("Late-Arriving-Data Monitor", Job({'failure.emails': 'camus@shopify.pagerduty.com',
```
Can we alert only acquisition @drdee ?

dterror-zz committed:
Adding LateArrivingDataMonitor to camus-shopify

After each Camus executions, this job will check, for all the topics
dropped, if anything was dropped in an old - already marked as stable -
folder. If that's the case (and thus we have late-arriving data), it
will fail. We'll then set alerting on the status of this job.

dterror-zz committed:
Fixing tests

Since Git messed with the mtime of the files, I had to go the explicit route, which is also a better route.

dterror commented:
@drdee :ship: ?

drdee commented:
Discussed this off-line, :ship:

yagnik commented:
Have we looked at the new importers of confluent, do they handle this scenario better ?

dterror commented:
We haven't looked extensibly, we'll be able to take them for a spin once we updated the cluster.

How are you doing, man?

yagnik commented:
I really feel that given the cheap cost of storage pretty much all cluster updates shuold move to coreos style, aka update on different partition and swap including all the configs.  (partially unrelated rant)

I'm doing well! how's everything there ?  I was chatting with @drdee yesterday and he was telling me about the beautiful land of data and how instafacts is the seed to the new world order. You guys need to really rename instafacts, it's a shit name

dterror commented:
haha you're absolutely right, but now everyone's using it and we're stuck with this.

Are you working with Coreos stuff? I'd love to hear that, we have to chat sometime outside of Github comments.
